docs/content/dagster-cloud/getting-started.mdx

```diff
@@ -1,7 +1,198 @@
 ---
-title: "Getting started with Dagster Cloud | Dagster Docs"
+title: Getting started with Dagster Cloud | Dagster Cloud
 
-noindex: True
+platform_type: "cloud"
 ---
 
-# Getting started with Dagster Cloud
+# Getting Started with Dagster Cloud
+
+<Note>This guide is applicable to Dagster Cloud.</Note>
+
+Welcome to Dagster Cloud!
+
+In this guide, we'll give you everything you need to get up and running with Dagster Cloud. We'll walk you through setting up your Dagster Cloud account, configuring deployment, loading your code, launching your first run, and configuring continuous integration.
+
+---
+
+## Prerequisites
+
+To complete the steps in this guide, you'll need access to your Dagster code.
+
+You can either use your own code or our [sample code (`dagster-cloud-examples`)](https://github.com/dagster-io/dagster-cloud/tree/main/dagster-cloud-examples). This guide uses the `dagster-cloud-examples` Python package.
+
+---
+
+## Step 1: Create a Dagster Cloud account and organization
+
+If you don't already have a Dagster Cloud account, [sign up for one](https://dagster.cloud/) before continuing.
+
+You'll also be asked to create a **Dagster Cloud organization.** When your organization is created, a single instance deployment, named `prod`, will also be created.
```

Comment:
s/"a single instance deployment"/"a single deployment"? Do we need to define what a deployment is here?

Reply:
I wondered that too. I don't want to get too in the weeds, but perhaps a brief description or at least linking out to a doc about (instance) deployments would work.